google-http-client/src/main/java/com/google/api/client/http/HttpTransport.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2010 Google Inc.
+ * Copyright (c) 2023 Google Inc.
```

Comment:
We don't want to update the year on new changes, see go/copyright#the-year

Reply:
done.